factcast-schema-registry-cli/src/main/kotlin/org/factcast/schema/registry/cli/commands/Build.kt

```diff
@@ -22,26 +22,36 @@ import javax.inject.Inject
 import kotlin.system.exitProcess
 
 @Command(
-    name = "build",
-    mixinStandardHelpOptions = true,
-    description = ["Validates and builds your registry"]
+        name = "build",
+        mixinStandardHelpOptions = true,
+        description = ["Validates and builds your registry"]
 )
 class Build : Runnable {
     @Option(names = ["-p", "--base-path"], description = ["The directory where your source files live"])
     var basePath: String = Paths.get(".").toString()
 
     @Option(names = ["-o", "--output"], description = ["Output directory of the registry"])
-    var outputPath: String = Paths.get(".", "output").toString()
+    var outputPath: String = Paths.get(".", "target/output").toString()
```

Comment:
Im not a big fan of `target/output` because the standard use case is without a maven/gradle project. Mybe we should go for something like `public/` (to stay in the hugo world)

Reply:
then it would be public/public, right ? :D

Reply:
That is....true 😂
Maybe we should work with a tmp folder and move whatever there is (hugo template when skipHugo is provided or the actual page) to `outputPath`? 🤔